manticore/platforms/linux.py

```diff
@@ -33,10 +32,13 @@
 from ..native.state import State
 from ..platforms.platform import Platform, SyscallNotImplemented, unimplemented
 
-from typing import Any, Dict, IO, List, Optional, Set, Tuple, Union
+from functools import wraps
+from typing import Any, Callable, cast, Deque, Dict, IO, List, Optional, Set, Tuple, TypeVar, Union
```

Comment:
Some of these are unnecessary, since there are no other changes in this file

Reply:
Good eye!  Whoops :|

One of these days we should consider turning on something like `isort`, or an unused import checker, as a lint step in our CI.